Sources/Sugar/String+Sugar.swift

```diff
@@ -1,13 +1,14 @@
 import Foundation
 
 extension String {
-	public func matchesRegex(_ regex: String) -> Bool {
+	public func matchesRegex(_ regex: String) -> Bool  {
 		return range(of: regex, options: .regularExpression, range: nil, locale: nil) != nil
 	}
 
-	public init(staticString: StaticString) {
+	public init(staticString : StaticString) {
 		self = staticString.withUTF8Buffer {
 			String(decoding: $0, as: UTF8.self)
 		}
 	}
 }
+
```

Comment:
<!--
  1 failure:  Files should have...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/Sugar/String+Sugar.swift;
  Line: 14;
-->
- :no_entry_sign: Files should have a single trailing newline. (`trailing_newline`)